tests/test_board.py

```diff
@@ -18,40 +18,64 @@ def test_board_has_nine_spaces_by_default(board):
 
 def test_can_occupy_a_space(board, space):
     board.place_marker(space, 'x')
+
     assert board.get_spaces()[space] == 'x'
 
 
-def test_returns_false_when_no_spaces_are_occupied(board):
+def test_board_is_not_full_when_no_spaces_are_occupied(board):
     assert board.is_full() is False
 
 
-def test_returns_false_when_board_is_not_full(board):
+def test_board_is_not_full_when_only_some_spaces_are_occupied(board):
     num_of_spaces_minus_one = len(board.get_spaces()) - 1
-    for _ in range(num_of_spaces_minus_one):
-        board.place_marker(_, 'x')
+    for space in range(num_of_spaces_minus_one):
+        board.place_marker(space, 'x')
+
     assert board.is_full() is False
 
 
-def test_returns_true_when_all_spaces_are_occupied(board):
-    for _ in range(len(board.get_spaces())):
-        board.place_marker(_, 'x')
+def test_board_is_full_when_all_spaces_are_occupied(board):
+    for space in range(len(board.get_spaces())):
+        board.place_marker(space, 'x')
+
     assert board.is_full() is True
 
 
-def test_returns_true_when_a_space_exists_and_is_unoccupied(board):
-    assert board.is_available_space(0) is True
+def test_a_space_is_available_when_it_exists_and_is_unoccupied(board):
+    assert board.is_valid_space(0) is True
 
 
-def test_returns_false_when_a_space_exists_but_is_occupied(board):
+def test_a_space_is_unavailable_when_it_exists_but_is_occupied(board):
     board.place_marker(0, 'x')
-    assert board.is_available_space(0) is False
 
+    assert board.is_valid_space(0) is False
```

Comment:
I disagree with this statement. `0` is a valid space. It's not available to be played on, but it's still valid.